### ISSUE TITLE
Link missing a text alternative (thumbnails on search results page)

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5344]
Sites identified: Archivo, IJCCR
Page(s) identified: 

- search results page (poster image links through to media object page)

WCAG compliance level: A

Description:

Links should always have a text alternative. The link text can be inserted into the HTML — it does not need to be visible to pass this check.

Links represented by images or icons can be problematic for visitors who are blind or have low vision.

The text alternative should describe the link's purpose — or what visitors can expect to find after they click it.

See more details at [SiteImprove example](https://my2.siteimprove.com/Inspector/1166400/A11Y/Page?pageId=42867640400&impmd=DB21396D67119C3E30B314CCF5B34C39&conf=a+aa+aria+si&wcag=twopointone&decision=dismissed+falsePositive+none#/sia-r11/failed/SlZ4aWRIcUM=;MmMxYzlkZTQ=)

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBcEZnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--990b76ab471db5b3a27adbd8eb231f6f68574543/image.png)

Note: although the scanner only identified this issue in Archivo, it's likely also a problem in MCO.

# Done Looks Like
- [ ] This link is hidden using aria (following the YouTube pattern) - screen readers and tab order for keyboard interaction should skip the link.

**Comment by joncameron** [comment]
We could hide the link (presentation role or hidden) for this image. Does it make sense to have both present to screen readers?

**Comment by charumitraravi** [comment]
QA verified.
